smart_contracts/contracts/test/do-nothing-stored-upgrader/src/main.rs

```diff
@@ -24,6 +24,7 @@ const ARG_PURSE_NAME: &str = "purse_name";
 
 #[no_mangle]
 pub extern "C" fn delegate() {
+    let _named_keys = runtime::list_named_keys();
```

Comment:
Should comment why this is necessary while the contract is still named "do-nothing".  Same for other two similar contracts.

Reply:
should rename the do-nothing family of contracts to "do-nothing-useful..." or "exercise-lower-gas-bounds..." or something like that

Reply:
I agree - I'll address it adequately in a separate ticket